charts/_versions.tpl

```diff
@@ -58,3 +58,7 @@ scheduling.k8s.io/v1alpha1
 {{- define "hpaversion" -}}
 autoscaling/v2beta1
 {{- end -}}
+
+{{- define "initializeradmissionregistrationversion" -}}
```

Comment:
Rename it to `admissionregistrationversion`. I don't think it will be different for `webhooks` and `initializer` in future.